msteams-platform/concepts/build-and-test/test-app-overview.md

```diff
@@ -25,6 +25,9 @@ Test your bot locally. For more information, see [debug your bot locally with an
 
 To view the console logs, view or modify html, css, and network requests during runtime, add breakpoints to your JavaScript code, and perform interactive debugging access the DevTools. For more information, see [access the DevTools for Teams tabs](~/tabs/how-to/developer-tools.md).
 
+> [!NOTE]
+> Debugging is supported in desktop, web, and android devices but not in iOS devices.
```

Comment:
@v-npaladugu , it is supported `on` the devices and not `in` them. 
Also, it should be `Andriod`.